ZipUtilities/NSStream+NOZAdditions.m

```diff
@@ -49,6 +49,14 @@ - (nonnull instancetype)initWithInputStream:(nonnull NSInputStream *)stream
 - (nonnull instancetype)initWithData:(nonnull NSData *)data NS_UNAVAILABLE;
 - (nullable instancetype)initWithURL:(nonnull NSURL *)url NS_UNAVAILABLE;
 
+// Undocumented CFReadStream methods
```

Comment:
What is this about?

Reply:
There are implementations of these below in the file. Nothing in the project calls those methods and they're in a section preceded with this:

`#pragma mark Undocumented CFReadStream methods`

So it seems like these are overrides of private methods from the parent class (and therefore ineligible for direct dispatch).